mod_ci/controllers.py

```diff
@@ -570,20 +590,16 @@ def start_ci():
         elif event == "issues":
             issue_data = payload['issue']
             issue = Issue.query.filter(Issue.issue_id == issue_data['number']).first()
+            issue_title = issue_data['title']
+            issue_id = issue_data['number']
+            issue_author = issue_data['user']['login']
+            issue_body = issue_data['body']
+            # Send Email to the Mailing List using the Mailer Module and Mailgun's API
+            inform_mailing_list(issue_id, issue_title, issue_author, issue_body)
```

Comment:
This will be sent every time a change is made to the issue. I'd wrap this function call in an if that checks if the issue was opened :)

Reply:
Oh shoot. I forgot that